src/gcp/cloudscheduler.ts

```diff
@@ -33,6 +33,13 @@ export function deleteJob(name: string): Promise<void> {
   });
 }
 
+export function getJob(name: string): Promise<any> {
```

Comment:
Please add jsdocs to all these public functions - that way it's clear what is returned and on what conditions errors are thrown. Also, would be nice to have a test for this.